website/settings/defaults.py

```diff
@@ -1890,3 +1890,6 @@ class CeleryConfig:
 VERIFY_CHRONOS_SSL_CERT = not DEV_MODE
 # Maximum minutes we allow ChronosSubmission status to be stale (only update when user is requesting it)
 CHRONOS_SUBMISSION_UPDATE_TIME = timedelta(minutes=5)
+
+DS_METRICS_OSF_TOKEN = None
+DS_METRICS_BASE_FOLDER = None
```

Comment:
Probably no. There's no reason local developers will need this unless they're explicitly debugging this management command, and then they'll know the name of the settings because they'll be right here.